exercises/K-functions-parameters/exercise.js

```diff
@@ -1,5 +1,5 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+function multiply(num1, num2) {
   // Calculate the result of the function and return it
```

Comment:
Inside the function the calculation needs to occur and answer be returned.
return num1 * num2